### PR TITLE
rel: Prep for release of v1.20

### PR DESCRIPTION
**Pull request by kentquirk**
Final doc tweaks for release of 1.20. 

Reviewers: should I add some subheadings to the summary?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+- When refinery shuts down, it will try to remove itself from the peers list, which should shorten the time of instability in the cluster.
+- The algorithm controlling how traces are distributed to nodes in the cluster has been revamped so that traces are much more likely to stay on the same node during reconfiguration. Set the [Peer Management](https://github.com/honeycombio/refinery/blob/main/config_complete.toml#L269) `Strategy` to `hash` to enable it.
+- A [Stress Relief](https://github.com/honeycombio/refinery/blob/main/config_complete.toml#L493) system has been added. When properly configured, it tracks refinery's load, and if it gets in danger of instability, switches into a high-performance mode designed to relieve stress on the system. When Activated, refinery samples spans at a deterministic rate and will continue doing so until the load is reduced. It also indicates which of its configuration values is most under stress, which should help tune it.
```
Comment (cartermp) [inline CHANGELOG.md:11]: I recommend mentioning the metric to monitor for this and its default value for when Stress Relief is activates.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+- Several new metrics have been added and the internal metrics system has been unified. This is to support the stress relief system.
```
Comment (cartermp) [inline CHANGELOG.md:12]: Which new metrics?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+- Dry Run mode no longer sets the Sample Rate, which means that Honeycomb queries will be accurate in this mode.
```
Comment (cartermp) [inline CHANGELOG.md:13]: Recommend mentioning the additional field it adds instead.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,67 @@
...
+- More Redis configuration is available to make it possible to do blue-green deployments with a single Redis instance.
```
Comment (cartermp) [inline CHANGELOG.md:14]: Which config value(s)?

**Review by cartermp — COMMENTED** [review]
Looks pretty good overall, but I think a few more additions will help people figure out what to do. Of course docs/samples should also reflect it, but I know some folks really do refer to changelog entries as a first source of what things are.

Additionally, since several of the enhancements here are settings ... we should probably file an issue tracking if/when it gets enabled by default? For example, is there a reality when the cuckoo cache will be a default behavior? If so, let's get it into issues.

**Comment by kentquirk** [comment]
@cartermp - almost all your comments are in the general form of "I want more!" 

I'm going to write a long, chatty RELEASE_NOTES.1.20.md document (coming soon), and also keep with the usual tradition of Honeycomb's `CHANGELOG.md` which is basically a list of the PRs in the release.  

In terms of when things will become defaults, the answer is "not until a 2.0 release" -- which we hope to do soon.

**Review by cartermp — APPROVED** [review]
This is awesome.

**Pull request merged**